extensions/wikia/UserLogin/js/UserLoginAjaxForm.js

```diff
@@ -87,8 +96,15 @@
 	 * @param {string} html
 	 */
 	UserLoginAjaxForm.prototype.retrieveTemplateCallback = function (html) {
-		var content = $('<div>').hide().append(html),
-			form = this.form;
+		var content, form;
+
+		if (typeof this.options.retrieveTemplateCallback === 'function') {
```

Comment:
Yes, I was thinking about something like this - just to check if there is anything - cos it's optional parameter.
On the other hand you could create a default empty function, and always call it - but all this function should be changed then because of that return below.

And don't worry - it's a good code. I just have different view on it and can't refrain from sharing it :)